MetaMask/metamask-extension
# UX: Send Flow: Place Amount field in content

## **Description**

Brian did a great job creating the `AssetPicker` but we need to shift it into just the content view (i.e. not show it until the user has picked a recipient).  This PR accomplishes that, with a few cleanups as well.

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Go to the new send flow
2. Don't see the AssetPicker
3. Choose a recipient
4. See the AssetPicker

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="402" alt="SCR-20231107-pudu" src="https://github.com/MetaMask/metamask-extension/assets/46655/a5cd9972-5e1a-40a6-81d8-874fffcb3950">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
